solr/modules/llm/src/java/org/apache/solr/llm/texttovector/model/SolrTextToVectorModel.java

```diff
@@ -63,12 +64,21 @@ public static SolrEmbeddingModel getInstance(
       var builder = modelClass.getMethod("builder").invoke(null);
       if (params != null) {
         /**
```

Comment:
This isn't javadoc; don't use double-asterisk

Reply:
my bad, it's apparent I don't do comments very much, fixed!